apps/volontulo/templates/offers/offer_form.html

```diff
@@ -5,73 +5,105 @@
 {% block content %}
     <form id="offer_form" class="form-horizontal" method="post" action="{{ request.get_full_path }}" role="form" enctype="multipart/form-data">
         {% csrf_token %}
-        <h2>{{ offer.id|yesno:"Edycja,Tworzenie" }} oferty dla wolontariuszy</h2>
+        <h2>{{ offer.id|yesno:"Edycja,Tworzenie" }} oferty</h2>
+
         {% if user.userprofile.is_administrator %}
         <div class="form-group form-group-sm">
-            <label class="col-xs-offset-6 col-xs-2 control-label" for="{{ offer_form.status.id_for_label }}">Status</label>
+            <label class="col-xs-offset-6 col-xs-2 control-label" for="{{ form.status.id_for_label }}">Status</label>
             <div class="col-xs-4">
-                <select name="status_old" class="form-control" id="{{ offer_form.status_old.id_for_label }}">
-                    {% for status, label in statuses.items %}
-                        {% if status == request.POST.offer|default:offer.status_old %}
-                        <option selected="selected" value="{{ status }}">{{ label }}</option>
-                        {% else %}
-                        <option value="{{ status }}">{{ label }}</option>
-                        {% endif %}
-                    {% endfor %}
+                <select name="status" class="form-control" id="{{ form.status.id_for_label }}">
+                    <option value="">-- Zmień status --</option>
```

Comment:
Shouldn't this be "Nieopublikowane"? Coz someone would like to take offer back to such state instead of "Odrzuć" it. But that could be a different task. 
The functionality appears to be okey